src/plugins/chart_expressions/expression_xy/common/expression_functions/xy_vis_fn.ts

```diff
@@ -127,6 +129,7 @@ export const xyVisFn: XyVisFn['fn'] = async (data, args, handlers) => {
       args: {
         ...restArgs,
         layers,
+        minBarHeight: args.minBarHeight ?? 1,
```

Comment:
Not sure how this default relates to the those in `x-pack/plugins/lens/public/visualizations/xy/to_expression.ts`